packages/views.vite-plugin/README.md

```diff
@@ -0,0 +1,100 @@
+# @osdk/views-manifest-vite-plugin
+
+This is a vite plugin that will automatically discover `*.config.(j|t)s` files that are imported into entrypoint files and generates a `.palantir/view-config.json` file for Foundry to read from. This allows developers to write out their configuration for custom views in Foundry in a single, type-safe file and not worry about generating a manifest file needed for publishing new versions of the view within Foundry.
+
+## Usage
+
+To use the plugin, add it to the `plugins` list of your [vite configuration](https://vitejs.dev/config/):
+
+```js
+import viewManifestPlugin from "@osdk/views-manifest-vite-plugin";
+import { defineConfig } from "vite";
+
+export default defineConfig({
+  plugins: [viewManifestPlugin()],
+  // Rest of configuration
+});
+```
+
+## Defining configuration
+
+The `@osdk/views-client` package exports a `ViewConfig` type so that you can define a configuration object in the shape that this plugin requires. You can define your config like so:
+
+```js
+// main.config.ts
```

Comment:
How does this relate to the dedicated parameters file I think you had before? Does it replace it?

Reply:
yes, it replaces it. I just renamed it because i realized it has events and parameters, so calling it `.parameters` felt off